docs/book/src/getting-started.md

```diff
@@ -93,13 +93,7 @@ it into a management cluster using `clusterctl`.
 
 ### Deploy with Cloud Contoller manager
 
-   1. To deploy VPC workload cluster with [IBM cloud controller manager](/topics/vpc/load-balancer.html), set the `PROVIDER_ID_FORMAT` environmental variable.
```

Comment:
<img width="794" alt="Screenshot 2023-11-27 at 7 03 18 PM" src="https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/assets/30797554/0b83a696-7fc0-428d-8dab-114f61c9a87b"> 
From preview, please re-check the formatting.